mmv1/templates/terraform/examples/firestore_index_basic.tf.erb

```diff
@@ -1,45 +1,17 @@
-resource "google_project" "project" {
-  project_id = "<%= ctx[:vars]['project_id'] %>"
-  name       = "<%= ctx[:vars]['project_id'] %>"
-  org_id     = "<%= ctx[:test_env_vars]['org_id'] %>"
-}
-
-resource "time_sleep" "wait_60_seconds" {
-  depends_on = [google_project.project]
-
-  create_duration = "60s"
-}
-
-resource "google_project_service" "firestore" {
-  project = google_project.project.project_id
-  service = "firestore.googleapis.com"
-
-  # Needed for CI tests for permissions to propagate, should not be needed for actual usage
-  depends_on = [time_sleep.wait_60_seconds]
-}
-
 resource "google_firestore_database" "database" {
-  project     = google_project.project.project_id
-  name        = "(default)"
+  project     = "<%= ctx[:test_env_vars]['project_id'] %>"
```

Comment:
I'm a little surprised this works given that you removed the `test_env_vars` section from the YAML

Reply:
I'm guessing the keys may just always be present?

The only combo I tried that didn't work was using [:vars]['project_id'], which failed because it used a test project that I don't have access to.

I cleaned this up so the yaml and erb files _should_ align. (but please double check me)